test/mongodb.test.js

```diff
@@ -308,6 +312,142 @@ describe('mongodb connector', function() {
       }
     );
 
+    modelWithDecimalArray = db.define('modelWithDecimalArray', {
```

Comment:
AFAICT, each of these models are used by a single test only. Please move their definitions to the tests that are using them, it will make the tests easier to read and also will avoid temptation to add unrelated properties to these models in the future.